<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="TabRec">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>611</width>
      <height>191</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <column>
     <property name="text">
      <string>device_id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p0</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p7</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="TabInit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>151</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <column>
     <property name="text">
      <string>device_id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p0(device_type)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p7</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="BtInit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Initialize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtRec">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Receive_Start/Stop</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="TabSet">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>611</width>
      <height>151</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <column>
     <property name="text">
      <string>device_id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p0</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p7</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="BtSet">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>210</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditPortSend">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>190</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>200</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SendPort</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditPortRec">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>360</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>RecPort</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>370</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RecPort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtAddInit">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtDelInit">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtAddSet">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>230</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtDelDel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>260</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Del</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
